tests/parametric/test_trace_sampling.py

```diff
@@ -49,9 +48,9 @@ class Test_Trace_Sampling_Basic:
     def test_trace_sampled_by_trace_sampling_rule_exact_match(self, test_agent, test_library):
         """Test that a trace is sampled by the exact matching trace sampling rule"""
         with test_library:
-            with test_library.start_span(name="web.request", service="webserver"):
+            with test_library.start_span(name="web.request", service="webserver") as span:
                 pass
-        span = find_span_in_traces(test_agent.wait_for_num_traces(1), Span(name="web.request", service="webserver"))
+        span = find_span_in_traces(test_agent.wait_for_num_traces(1), span.trace_id, span.span_id)
```

Comment:
Could we use `find_only_span()` here (and other tests below) since we expect only one span?

Reply:
I added find_only_span after making this change and I was too lazy to revert it. I will fix this